pkg/compiler/debug.go

```diff
@@ -267,6 +268,59 @@ func (d *DebugParam) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+// ToManifestParameter converts DebugParam to manifest.Parameter
+func (d *DebugParam) ToManifestParameter() (manifest.Parameter, error) {
```

Comment:
`DebugParam` was created when there was no `manifest` thing. Now we can use `manifest.Parameter`/`Method`/`Event` from the start. May be even whole `DebugInfo` can be replaced with `Manifest`.

Reply:
We can't, because they're slightly different in the way they should be serialized into JSON (e.g. `manifest.Method.ReturnType` should be serialized as `returnType` vs `return-type` of `DebugInfo.Method.ReturnType`). See https://github.com/ngdseattle/design-notes/blob/master/NDX-DN11%20-%20NEO%20Debug%20Info%20Specification.md#v11-format for details.